cli/types/index.d.ts

```diff
@@ -2934,6 +2942,130 @@ declare namespace Cypress {
     (fn: (currentSubject: Subject) => void): Chainable<Subject>
   }
 
+  /**
+   * These events come from the application currently under test (your application). These are the most useful events for you to listen to.
+   * @see https://docs.cypress.io/api/events/catalog-of-events.html#App-Events
+   */
+  interface Actions {
+    /**
+     * Fires when an uncaught exception occurs in your application.
+     * Cypress will fail the test when this fires.
+     * Return `false` from this event and Cypress will not fail the test. Also useful for debugging purposes because the actual `error` instance is provided to you.
+     * @example
+     * // likely want to do this in a support file
+     * // so it's applied to all spec files
+     * // cypress/support/index.js
+     *
+     * Cypress.on('uncaught:exception', (err, runnable) => {
+     *   // returning false here prevents Cypress from
+     *   // failing the test
+     *   return false
+     * })
+     * @see https://docs.cypress.io/api/events/catalog-of-events.html#App-Events

```

Comment:
Should there be linkability to individual events? Right now this will jump to the table of actions. Also I don't know if that is the official URL. I noticed the Cypress team use a redirect URL for Cypress commands. I couldn't find anything that matched that pattern (`on.cypress.io/...`) that redirected to the correct page. Feel free to suggest a redirect URL.